keyboards/id80/info.json

```
@@ -8,6 +8,98 @@
+                {"label":"F2", "x":2.25, "y":0}, 
+                {"label":"F3", "x":3.25, "y":0}, 
+                {"label":"F4", "x":4.25, "y":0}, 
+                {"label":"F5", "x":5.5, "y":0}, 
+                {"label":"F6", "x":6.5, "y":0}, 
+                {"label":"F7", "x":7.5, "y":0}, 
+                {"label":"F8", "x":8.5, "y":0}, 
+                {"label":"F9", "x":9.75, "y":0}, 
+                {"label":"F10", "x":10.75, "y":0}, 
+                {"label":"F11", "x":11.75, "y":0}, 
+                {"label":"F12", "x":12.75, "y":0}, 
+                {"label":"Fn", "x":14, "y":0}, 
+                {"label":"Insert", "x":15.25, "y":0}, 
+
+                {"label":"`", "x":0, "y":1.25}, 
+                {"label":"1", "x":1, "y":1.25}, 
+                {"label":"2", "x":2, "y":1.25}, 
+                {"label":"3", "x":3, "y":1.25}, 
+                {"label":"4", "x":4, "y":1.25}, 
+                {"label":"5", "x":5, "y":1.25}, 
+                {"label":"6", "x":6, "y":1.25}, 
+                {"label":"7", "x":7, "y":1.25}, 
+                {"label":"8", "x":8, "y":1.25}, 
+                {"label":"9", "x":9, "y":1.25}, 
+                {"label":"0", "x":10, "y":1.25}, 
+                {"label":"-", "x":11, "y":1.25}, 
+                {"label":"=", "x":12, "y":1.25}, 
+                {"label":"Backspace", "x":13, "y":1.25, "w":2}, 
+                {"label":"Home", "x":15.25, "y":1.25}, 
+
+                {"label":"Tab", "x":0, "y":2.25, "w":1.5}, 
+                {"label":"Q", "x":1.5, "y":2.25}, 
+                {"label":"W", "x":2.5, "y":2.25}, 
+                {"label":"E", "x":3.5, "y":2.25}, 
+                {"label":"R", "x":4.5, "y":2.25}, 
+                {"label":"T", "x":5.5, "y":2.25}, 
+                {"label":"Y", "x":6.5, "y":2.25}, 
+                {"label":"U", "x":7.5, "y":2.25}, 
+                {"label":"I", "x":8.5, "y":2.25}, 
+                {"label":"O", "x":9.5, "y":2.25}, 
+                {"label":"P", "x":10.5, "y":2.25}, 
+                {"label":"[", "x":11.5, "y":2.25}, 
+                {"label":"]", "x":12.5, "y":2.25}, 
+                {"label":"Enter", "x":13.5, "y":2.25, "w":1.5, "h":2},
+                {"label":"Delete", "x":15.25, "y":2.25}, 
+
+                {"label":"Caps Lock", "x":0, "y":3, "w":1.75},
+                {"label":"A", "x":1.75, "y":3.25},
+                {"label":"S", "x":2.75, "y":3.25},
+                {"label":"D", "x":3.75, "y":3.25},
+                {"label":"F", "x":4.75, "y":3.25},
+                {"label":"G", "x":5.75, "y":3.25},
+                {"label":"H", "x":6.75, "y":3.25},
+                {"label":"J", "x":7.75, "y":3.25},
+                {"label":"K", "x":8.75, "y":3.25},
+                {"label":"L", "x":9.75, "y":3.25},
+                {"label":";", "x":10.75, "y":3.25},
+                {"label":"'", "x":11.75, "y":3.25},
+                {"label":"ISO #", "x":12.75, "y":3.25},
+
```

Move the Enter key's object to match the updated `id80.h`.

(3/9)

```suggestion
                {"label":"Enter", "x":13.5, "y":2.25, "w":1.5, "h":2},

```